packages/connection/src/class-manager.php

```diff
@@ -519,9 +519,9 @@ public function is_active() {
 	 * @return bool
 	 */
 	public function is_registered() {
-		$blog_id   = \Jetpack_Options::get_option( 'id' );
-		$has_token = $this->is_active();
-		return $blog_id && $has_token;
+		$has_blog_id    = ! ! \Jetpack_Options::get_option( 'id' );
+		$has_blog_token = ! ! $this->get_access_token( false );
```

Comment:
That may be a stupid question, but is there a reason to use `!!` over `(bool)` in scenarios like this one? I find the latter more readable, but maybe I'm missing something here.

Reply:
Fair point, `(bool)` is probably better.